gradle/libs.versions.toml

```diff
@@ -1,7 +1,9 @@
 [versions]
 agp = "8.4.0"
 datastorePreferences = "1.1.1"
+graphSupport = "1.3.1"
 jgraphtCore = "1.5.2"
+jgraphtIo = "1.5.2"
```

Comment:
if you don't mind, I'd really prefer keeping just one version variable for the jgrapht library (then using it for both jgrapht-io and jgrapht-core's version-refs). My experience in industry was that it's borrowing trouble to use different versions for two different maven artifacts that are effectively part of the same library.